src/index.jsx

```diff
@@ -73,13 +74,17 @@ apiClient.ensurePublicOrAuthenticationAndCookies(
   (accessToken) => {
     const { store, history } = configure();
 
-    ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
-
     if (accessToken) {
+      ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
       identifyAuthenticatedUser(accessToken.userId);
+      sendPageEvent();
     } else {
+      // This should never happen, but it does sometimes.
+      // Add logging to learn more.
+      ReactDOM.render(<ErrorPage />, document.getElementById('root'));
+      NewRelicLoggingService.logError('Empty accessToken returned from ' +
+        'ensurePublicOrAuthenticationAndCookies callback.');
```

Comment:
Is there anything else useful we can add about the state?

Reply:
@robrap Is there anything useful here? I can't think of anything.

Reply:
@albemarle: FYI: We've got other related frontend-auth code changes to help log and track this down.
@abutterworth: We should think about the right interface for ensurePublicOrAuthenticationAndCookies though.  It seems wrong to be calling this callback in this error condition.